packages/fiori/src/Timeline.js

```diff
@@ -30,6 +36,19 @@ const metadata = {
 			individualSlots: true,
 		},
 	},
+	properties: /** @lends  sap.ui.webcomponents.fiori.Timeline.prototype */ {
+		/**
+		 * Select orientation.
+		 *
+		 * @type {String}
+		 * @defaultvalue "Vertical"
+		 * @public
+		 */
+		axisOrientation: {
+			type: String,
```

Comment:
This property can be enum with two options. With this approach the variable ```AXIS_ORIENTATION``` will not be needed anymore. 
(In the enum, both values should start with a capital letter)

Reply:
Done